lib/mandrill_dm/message.rb

```diff
@@ -64,6 +64,17 @@ def html
       mail.html_part ? mail.html_part.body.decoded : mail.body.decoded
     end
 
+    def template
```

Comment:
There's already a `template_name ` modifier in [actionmailer](http://api.rubyonrails.org/classes/ActionMailer/Base.html), what do you think we should do?